conda/recipes/mantid-developer/meta.yaml

```diff
@@ -0,0 +1,97 @@
+{% set version = load_setup_py_data(setup_file='../../../setup.py', from_recipe_dir=True).get('version', '0.0.0') %}
+
+package:
+  name: mantid-developer
+  version: {{ version }}
+
+requirements:
+  run:
+    - ccache
+    - cmake {{ cmake }}
+    - conda-wrappers # [win]
+    - doxygen {{ doxygen }}
+    - eigen {{ eigen }}
+    - euphonic {{ euphonic }}
+    - graphviz {{ graphviz }}
+    - gsl {{ gsl }}
+    - h5py
+    - hdf5 {{ hdf5 }}
+    - jemalloc {{ jemalloc }} # [linux]
+    - jsoncpp {{ jsoncpp }}
+    - lib3mf # [win]
+    - libboost-devel {{ libboost_devel }}
+    - libboost-python-devel {{ libboost_python_devel }}
+    - libopenblas {{ libopenblas }} # [osx or linux]
+    - librdkafka {{ librdkafka }}
+    - matplotlib {{ matplotlib }}
+    - muparser {{ muparser }}
+    - nexus {{ nexus }}
+    - ninja {{ ninja }}
+    - numpy>=1.24,<1.27 # This is intentionally different to conda_build_config.yaml, will address later.
+    - occt {{ occt }}
+    - pip {{ pip }}
+    - poco {{ poco }}
+    - psutil {{ psutil }}
+    - pycifrw
+    - pydantic
+    - pyqt {{ pyqt }}
+    - python-dateutil {{ python_dateutil }}
+    - python {{ python }}
+    - python.app # [osx]
+    - pyyaml {{ pyyaml }}
+    - qscintilla2 {{ qscintilla2 }}
+    - qt {{ qt_main }}
+    - qtconsole {{ qtconsole }}
+    - qtpy {{ qtpy }}
+    - qt-gtk-platformtheme # [linux]
+    - quasielasticbayes
+    - requests>=2.25.1
```

Comment:
And is this version also intentionally not put in the conda_build_config.yaml?

Reply:
From Tom: Requests is only found in the developer environment and not in any of the package environments. In future might be worth checking why this is (but out of scope for this PR)